### PR TITLE
Swagger endpoint

### PR DESCRIPTION
**Pull request by jporres1296**

### CODE REVIEW

File: docs/swagger.json [hunk 1]
```diff
@@ -1,532 +1,544 @@
...
     "license": {
       "name": "MPL-2.0",
       "url": "https://opensource.org/licenses/MPL-2.0"
     },
     "version": "0.0.1"
   },
-  "basePath": "/v1",
+  "servers": [
+    {
+      "url": "https://jon.io:8089/v1"
```
Comment (jjrodrig) [inline docs/swagger.json]: Please use https://dev.lamassu.io:8089/v1 as url

File: docs/swagger.json [hunk 2]
```diff
@@ -1,532 +1,544 @@
...
+    "/devices": {
+      "get": {
+        "tags": [
+          "Device Manager"
+        ],
+        "summary": "Get Devices",
+        "operationId": "GetDevice",
+        "parameters": [
+          {
+            "name": "Authorization",
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

File: docs/swagger.json [hunk 3]
```diff
@@ -1,532 +1,544 @@
...
+      },
+      "post": {
+        "tags": [
+          "Device Manager"
+        ],
+        "summary": "Create Device",
+        "operationId": "CreateDevice",
+        "parameters": [
+          {
+            "name": "Authorization",
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

File: docs/swagger.json [hunk 4]
```diff
@@ -1,532 +1,544 @@
...
+            "name": "deviceId",
+            "in": "path",
+            "description": "Device ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "Authorization",
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

File: docs/swagger.json [hunk 5]
```diff
@@ -1,532 +1,544 @@
...
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

File: docs/swagger.json [hunk 6]
```diff
@@ -1,532 +1,544 @@
...
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

File: docs/swagger.json [hunk 7]
```diff
@@ -1,532 +1,544 @@
...
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

File: docs/swagger.json [hunk 8]
```diff
@@ -1,532 +1,544 @@
...
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

File: docs/swagger.json [hunk 9]
```diff
@@ -1,532 +1,544 @@
...
```
Comment (jjrodrig) [inline docs/swagger.json]: Use security schemas for authorization

**Review by jjrodrig — CHANGES_REQUESTED** [review]

File: docs/swagger.json (hunk 1 above)
Comment (jporres1296) [reply docs/swagger.json]: In the last commit ("Swagger Auth OpenApi 3.0") are all changes made.

**Review by jjrodrig — APPROVED** [review]

**Pull request merged**